cmake/modules/CTags.cmake

```diff
@@ -3,10 +3,11 @@ find_program(CTAGS_EXECUTABLE ctags)
 function(add_tags name)
   cmake_parse_arguments(TAGS "" "SRC_DIR;TAG_FILE" "EXCLUDE_OPTS;EXCLUDES" ${ARGN})
   set(excludes ${TAGS_EXCLUDES})
+  find_package(Git)
   if(TAGS_EXCLUDE_OPTS)
     # always respect EXCLUDES_OPTS
     list(APPEND excludes ${TAGS_EXCLUDE_OPTS})
-  else()
+  elseif(Git_FOUND)
     # exclude the submodules under SRC_DIR by default
     execute_process(
       COMMAND git config --file .gitmodules --get-regexp path
```

Comment:
since we are using `find_package(Git)`, better off using `${GIT_EXECUTABLE}` here. see https://cmake.org/cmake/help/latest/module/FindGit.html

Reply:
fair point